packages/cli/test/unit/commands/build/index.test.ts

```diff
@@ -376,6 +376,9 @@ describe('build', () => {
     }
     expect(client.telemetryEventStore).toHaveTelemetryEvents([
       { key: 'flag:yes', value: 'TRUE' },
+      // from `vc env pull`
+      { key: 'argument:filename', value: '[REDACTED]' },
+      { key: 'option:environment', value: '[REDACTED]' },
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** We can remove these later after we resolve this command-to-command logic.

See: https://github.com/vercel/vercel/pull/12372/files‏